--- musl.mk	2026-05-11 07:18:11.612421805 +0000
+++ musl.mk	2026-05-11 07:20:15.202237872 +0000
@@ -4,8 +4,8 @@
 #
 ################################################################################
 
-MUSL_VERSION = 1.2.6
-MUSL_SITE = http://musl.libc.org/releases
+MUSL_VERSION = 1.1.24
+MUSL_SITE = http://www.musl-libc.org/releases
 MUSL_LICENSE = MIT
 MUSL_LICENSE_FILES = COPYRIGHT
 MUSL_CPE_ID_VENDOR = musl-libc
